README.md

```diff
@@ -1,409 +1,73 @@
-    - [x] room protocol routes
-      - [x] ping
-      - [x] room ready
-      - [x] match started
-      - [x] match ended
-      - [x] address polling
-    - [x] docs
-  - [x] [maestro-client](https://github.com/topfreegames/maestro-client)
+Maestro is an unified system that automatically scales game rooms regardless of the protocol (TCP, UDP). This system is related to a matchmaker but does not handle the specificities of a match such as how many players fit in a room. It only deals with high level room occupation, i.e. is the room occupied or available.
 
-- Milestone 2:
+## What does the Maestro do?
 
-  Goals:
-    - Create scheduler that scales down with the given policy.
-    - Update running scheduler.
-    - Monitor scheduler rooms metrics.
+Maestro is a game room scheduler that is composed by a controller, a watcher, a worker, an API and a CLI.
 
-  Tasks:
-  - [x] maestro-controller
-    - [x] scheduler
-        - [x] update running scheduler config
-          - [x] update config in database
-          - [x] update GRUs
-            - [x] launch new GRUs with the updates config
-            - [x] gracefully terminate GRUs running with old config
-    - [x] scheduler state
-      - [x] report room occupation metrics
-    - [x] scaling
-      - [x] remove GRUs
-  - [x] maestro-watcher
-    - [x] scale cluster
-      - [x] down
-  - [x] maestro-api
-    - [x] scheduler
-      - [x] update running scheduler config
-    - [x] get rooms metrics
-  - [x] maestro-cli
-    - [x] scheduler
-      - [x] create new scheduler with given config
-      - [x] update running scheduler config
-      - [x] delete scheduler
-      - [x] scheduler status
-    - [x] docs
-  - [ ] maestro-client
-    - [ ] cocos2dx support
-  - [ ] UI
-    - [ ] display rooms metrics
-    - [ ] scheduler
-      - [ ] create new scheduler with given config
-      - [ ] update running scheduler config
-      - [ ] delete scheduler
+## What problem is solved?
 
-## Doubts
+Maestro provides the ability for game teams to deploy Game Room Unities (GRUs) in an easy way. And the possibility to configure a more efficient allocation strategy to its GRUs. With that is expected that costs saved and less money lost with the game infrastructure.
 
-- ~~Can Kubernetes handle thousands of services?~~
-- How to manage different versions running at the same time? Will the matchmaker be responsible for it?
-- How to properly tune autoscaling policies?
+## Recommended Integration Phase: Alpha
 
-## Architecture Validation and Tests
+Usually, each game has its own time to include multiplayer feature, so generally is recommended to include it into the project during alpha.
+If multuiplayer GRUs is required during prototyping, this module should be included in the prototyping phase as well.
 
-### Validating Kubernetes performance
+## Dependencies
+Maestro depends only infrastructure services to operate, the most required is a Kubernetes cluster where Maestro resides.
```

Comment:
@arthur29 this section relates to other modules' dependencies only, you can mention, for example, that if the user opt to use events forwarding, an external matchmaking service dependency is required